assets/stylesheets/style.scss

```diff
@@ -3,12 +3,7 @@
 
 // Shared
 @import 'shared/reset';                        // css reset before the rest of the styles are defined
-@import 'shared/functions';                    // functions that we've used from Compass, ported over
-@import 'shared/functions/functions';          // sass functions for z-index, etc.
-@import 'shared/colors';                       // import all of our wpcom colors
-@import 'shared/typography';                   // all the typographic rules, variables, etc.
-@import 'shared/mixins/mixins';                // sass mixins for gradients, bordius radii, etc.
-@import 'shared/extends';                      // sass extends for commonly used styles
+@import 'shared/utils';                        // utilities that are used by all CSS but don't produce any code
```

Comment:
Grouping all these functions and stuff into one file because they will need to be used every time we split CSS into a new file.